src/rf/apps/core/models.py

```diff
@@ -177,11 +187,49 @@ def to_json(self):
             'tile_origin': self.tile_origin,
             'resampling': self.resampling,
             'transparency': self.transparency,
-            'status': self.status,
+            'status_created': (True if self.status_created is not None
+                               else False),
+            'status_upload_start': (True if self.status_upload_start
+                                    is not None else False),
+            'status_upload_end': (True if self.status_upload_end is not None
+                                  else False),
```

Comment:
The timestamps can be useful for debugging, but we essentially have them in rf-worker.log.